packages/@aws-cdk/aws-pipes-alpha/lib/inputTransformation.ts

```diff
@@ -0,0 +1,55 @@
+
+enum reservedVariables {
+  PIPES_ARN = '<aws.pipes.pipe-arn>',
+  PIPES_NAME = '<aws.pipes.pipe-name>',
+  PIPES_TARGET_ARN = '<aws.pipes.target-arn>',
+  PIPE_EVENT_INGESTION_TIME = '<aws.pipes.event.ingestion-time>',
+  PIPE_EVENT = '<aws.pipes.event>',
+  PIPE_EVENT_JSON = '<aws.pipes.event.json>',
+}
+
+type StaticString = string;
+// type JsonPath = `<$.${string}>`;
+type KeyValue = Record<string, string | reservedVariables>;
+type StaticJsonFlat = Record<string, StaticString | KeyValue>;
+type InputTransformJson = Record<string, StaticString | KeyValue | StaticJsonFlat>;
+
+type InputTransformationValue = StaticString | InputTransformJson;
+
+/**
+ * Transform or replace the input event payload
+ */
+export interface IInputTransformation {
+  /**
+   * Valid JSON text passed to the target.
+   */
+  inputTemplate: string;
```

Comment:
pubic readonly inputTemplate: string;
```suggestion
  readonly inputTemplate: string;
```